actinia_stac_plugin/api/stac.py

```diff
@@ -83,3 +83,12 @@ def get(self, collection_id):
         return make_response(module_list, 200)
 
 
+class StacCollectionList(ResourceBase):
+
+    def get(self): 
+        return ""
+
+class StacInstanceList(ResourceBase):
+
+    def get(self): 
+        return ""
```

Comment:
I think no new line at the end of a file is not the standard. Maybe you want to include a linting directly e.g. like in https://github.com/mundialis/actinia-module-plugin/blob/main/.github/workflows/flake8.yml. With exceptions as in https://github.com/mundialis/actinia-module-plugin/blob/main/.flake8, if needed.

Reply:
I think @mmacata wanted to work on this in #4

Reply:
And yes, having a newline at the end of the file would be my preference

Reply:
I talked to Jorge, he wants to learn it as well so he will look into it soon.